packages/e2e-cypress/src/helpers/cct-dev-server/index.ts

```diff
@@ -68,12 +122,10 @@ async function quasarWebpackConfig(quasarAppPackage: string) {
   return configEntries[0].webpack;
 }
 
-async function quasarViteConfig(quasarAppPackage: string) {
-  const { QuasarConfFile, ctx } = await quasarSharedConfig(
-    quasarAppPackage,
-    'vite',
-  );
-
+async function quasarViteConfig(
+  quasarAppPackage: string,
+  { QuasarConfFile, ctx }: { QuasarConfFile: any; ctx: any },
+) {
   const quasarConfFile = new QuasarConfFile({ ctx });
 
   const quasarConf = await quasarConfFile.read();
```

Comment:
Care as the `spa-config` import changed between the 2 versions, so you definitely need the `isOldVersion` here too